src/cli/cli_dataset.cpp

```diff
@@ -553,5 +596,69 @@ ThreadError Dataset::ProcessMgmtGetCommand(int argc, char *argv[])
     return error;
 }
 
+ThreadError Dataset::ProcessPSKc(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+    uint16_t length;
+
+    VerifyOrExit(argc > 0, error = kThreadError_Parse);
+
+    length = static_cast<uint16_t>((strlen(argv[0]) + 1) / 2);
+    VerifyOrExit(length <= OT_PSKC_MAX_SIZE, error = kThreadError_NoBufs);
+    VerifyOrExit(Interpreter::Hex2Bin(argv[0], sDataset.mPSKc.m8 + OT_PSKC_MAX_SIZE - length, length)
+                 == length, error = kThreadError_Parse);
+
+    sDataset.mIsPSKcSet = true;
+
+exit:
+    return error;
+}
+
+ThreadError Dataset::ProcessSecurityPolicy(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+    long value;
+
+    VerifyOrExit(argc > 1, error = kThreadError_Parse);
+
+    SuccessOrExit(error = Interpreter::ParseLong(argv[0], value));
+    sDataset.mSecurityPolicy.mRotationTime = static_cast<uint16_t>(value);
+    sDataset.mSecurityPolicy.mFlags = 0;
+
+    for (char *arg = argv[1]; *arg != '\0'; arg++)
+    {
+        switch (*arg)
+        {
+        case 'O':
```

Comment:
nit: we've been using lowercase for the flags, let's keep them consistent for now.